src/gameData/const_gui_ids.h

```diff
@@ -63,7 +63,8 @@ enum GUI_ID
     CGI_MISSION_STATEMENT,
     CGI_MAP_DEBUG,
     CGI_AI_DEBUG,
-    CGI_NEXT = CGI_MAP_DEBUG + 40
+    CGI_NEXT = CGI_MAP_DEBUG + 40,
+    CGI_MAP_GENERATOR
```

Comment:
This needs to be before NEXT and NEXT needs to be updated (someone forgot it already)